systemd/user.service

```diff
@@ -3,7 +3,7 @@ Description=The Lounge IRC client
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/lounge start
+ExecStart=THELOUNGE_HOME=~/.lounge /usr/bin/thelounge start
```

Comment:
Shouldn't home be ``/etc/lounge` here or the ENV removed?

Reply:
That's for the user service. Also this is a commit coming from #13 that I need to remove here

Reply:
This change doesn't make sense in that PR either, does it? Why are you changing it to home folder if it was in etc before.

Reply:
Because it's the user service and not the system service. Let's talk about this in the other pr

Reply:
Or I can open a new PR just for that. It's just outside the scope of this PR, but I needed the change to try making it work

Reply:
I'll clean up both PRs tomorrow